extract-headers.sh

```diff
@@ -207,6 +207,15 @@ check_header_exists bionic/libc/kernel/common/linux/sync.h && \
         bionic/libc/kernel/common/linux/sync.h \
         bionic/libc/kernel/common/linux/sw_sync.h
 
+check_header_exists bionic/libc/kernel/uapi/linux/sync.h && \
```

Comment:
This change is already in the modular layout: https://github.com/mer-hybris/droid-hal-device/commit/e0d7a4bfc9dc2a038103f727c20efcbb6f9761e1
So really advisable to migrate ;)